### PR TITLE
1472 selectable fields

### LINKED ISSUE #1472: FEAT: selectable fields in the table structure view
When coding, I often need to go find the exact name of a column in a table. Using Beekeeper, I open the "table structure" view. Ideally I want to be able to highlight a column name and copy it to clipboard. But I can't even highlight text fields in this view.

Would it be possible to make it happen?

### PR DESCRIPTION
**Pull request by not-night-but**
Fix #1472. Now this tooltip will show when a column name cannot be edited:
image

And the user can click the column name to copy it to clipboard

### CODE REVIEW

**Comment by rathboma** [comment]
Can we also add the `selected` style that we use for non-editable cells in the TableTable view?

Peek 2023-02-20 21-38

**Review by rathboma — CHANGES_REQUESTED** [review]
Looks good, but for consistency, lets add the cell-selected style we use in TableTable too.

File: apps/studio/src/components/tableinfo/TableSchema.vue [hunk 1]
```diff
@@ -417,6 +420,18 @@ export default Vue.extend({
         data: this.tableData,
         placeholder: "No Columns",
       })
+    },
+    columnNameCellClick(_e: any, cell: CellComponent) {
```
Comment (rathboma) [inline apps/studio/src/components/tableinfo/TableSchema.vue:424]: If you click away from the table does it unselect the cell? Take a look at table table for how to do that.

**Review by rathboma — CHANGES_REQUESTED** [review]
This needs to handle the case when someone clicks away from the cell, in that case it should de-select it. See how we do this in tabletable (we add a global click watcher).

**Comment by not-night-but** [comment]
@rathboma handling global clicks now

**Review by rathboma — APPROVED** [review]
Actual change looks good! Need to fix the weird other changes before merging though.

**Comment by not-night-but** [comment]
> The diff on the pr looks like it has a bunch of other changes in it? Can you merge in master and push up?
> 
> this might be a GitHub oddity

Actually looking at the commit history I'm super confused as to what's happened. I try to keep my branches up to date with main, something must've happened during one of the merges?

**Pull request merged**